cmp/grid/columns/Column.js

```diff
@@ -187,7 +186,15 @@ export class Column {
                 lockPinned: true, // Block user-driven pinning/unpinning - https://github.com/exhi/hoist-react/issues/687
                 pinned: this.pinned,
                 lockVisible: !gridModel.colChooserModel,
-                headerComponentParams: {gridModel, column: this}
+                headerComponentParams: {gridModel, hoistColumn: this},
```

Comment:
These `headerComponentParams` will actually get merged onto the params which ag-grid puts together for the header component, which already includes a `column` prop for the ag-grid Column object. Us adding our column here was causing it to be merged into the ag-grid Column object. Caused a lot of issues with the secondary columns created during pivoting, which isn't supported yet in this PR, but wanted to get this fix in asap.

Note the related changes in the ColumnHeader component.

Reply:
really nice fix.  just wonder if we want to use xhColumn -- has a nice symmetry with agColumn.